### DISCUSSION TITLE
How can I display a UIImage that includes an SFSymbol using the SwiftUI Image component?

### DISCUSSION THREAD

**Discussion by satoshinakamotofan** [community/community#135566] [Programming Help] [answered]
### Body

I have an image that displays its contents using UIImage objects passed in. Now, I need to include an SFSymbol in the image. I'm testing in a separate sample to see if it's possible to do this using a UIImage with the systemName, which corresponds to the SFSymbol name. However, the following code does not render any image:

```swift
struct ContentView: View {
    var body: some View {
        VStack {
            Image(uiImage: UIImage(systemName: "airplane.circle.fill")!.withRenderingMode(.alwaysOriginal))
                .resizable()
                .aspectRatio(contentMode: .fit)
                .frame(width: 150, height: 150)
        }
        .background {
            Color.black.ignoresSafeArea()
        }
    }
}
```

I understand that I could use the systemName initializer for Image with the symbol name, but unfortunately, that isn't feasible due to some constraints. Is there a way to achieve this? Any assistance would be greatly appreciated.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by everforyou** [comment, marked as answer]
You're displaying a black image on a black background.

To make the image visible, change `Color.black.ignoresSafeArea()` to a different color, such as `Color.white.ignoresSafeArea()`.

**Reply by satoshinakamotofan** [reply]
It works, thanks you!
